vpr/src/route/lookahead_profiler.h

```diff
@@ -0,0 +1,58 @@
+#ifndef VTR_LOOKAHEAD_PROFILER_H
+#define VTR_LOOKAHEAD_PROFILER_H
+
+#include <fstream>
+#include <thread>
+
+#include "connection_router_interface.h"
+#include "router_lookahead.h"
+#include "rr_graph_fwd.h"
+
+/**
+ * @brief A class which records information used to profile the router lookahead: most importantly,
+ * the actual cost (delay and congestion) from nodes to the sink to which they have been routed, as
+ * well as the lookahead's estimation of this cost.
+ */
+class LookaheadProfiler {
+  public:
+    LookaheadProfiler()
+        : is_empty(true) {}
```

Comment:
delete default constuctors and operators, assuming this class is not intented to be copied or moved